aws/resource_aws_route_table.go

```diff
@@ -353,6 +353,11 @@ func resourceAwsRouteTableUpdate(d *schema.ResourceData, meta interface{}) error
 				opts.NatGatewayId = aws.String(s)
 			}
 
+			// #69 - just do it! AND don't complain on subsequent runs that it needs to _change_ :/
```

Comment:
#69 is the github ticket that this PR resolves. I linked to that in an attempt to provide said context. however, I'm happy to delete this comment or change it to whatever you prefer.